src/functionalTest/java/uk/gov/hmcts/reform/laubackend/idam/serenityfunctionaltests/steps/BaseSteps.java

```diff
@@ -154,5 +154,22 @@ public String thenBadResponseIsReturned(Response response, int expectedStatusCod
         return TestConstants.SUCCESS;
     }
 
+    @Step("When the GET service is invoked")
+    public Response whenTheGetServiceIsInvoked(
+        String endpoint,
+        String serviceToken,
+        String authorizationToken,
+        Map<String, String> queryParams
+    ) throws JsonProcessingException {
+        return performGetOperation(endpoint, null, queryParams, serviceToken, authorizationToken);
+    }
 
+    @Step("Then unauthorised response is returned")
+    public String thenUnauthorizedResponseIsReturned(Response response, int expectedStatusCode) {
```

Comment:
You don't need this, you can re-use `thenBadResponseIsReturned(...)` method. Here, bad response has broad meaning that request hasn't been successful. Technically there is no bad response, there is a bad request. So in my opinion, both unauthorized and bad request can use one and the same method.

Reply:
The method is unused.

Reply:
Done